src/js/feedback/components/DefaultView.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function GiveUsFeedback({ feedbackButtonClicked }) {
```

Comment:
Should the file name and the function name match?

Reply:
Yeah that would probably be better.